### PR TITLE
Fix: Resendable paychecks to altinn if report is rejected

### LINKED ISSUE #267: Altinn level on received messages
In altinn request 4 we receive  this xml +++

``` xml
<mottattLeveranse>
  <altinnReferanse>ARxxxxxxxxx</altinnReferanse>
  <meldingsId>report_for_xxxxxxxxx_at_1473669538</meldingsId>
  <tidsstempelFraAltinn>2016-09-12T10:41:59.707+02:00</tidsstempelFraAltinn>
  <mottakstatus>avvist</mottakstatus>
  <kildesystem>LODO</kildesystem>
</mottattLeveranse>
```

If the XML look like this the message was rejected. If it look like this then the salaries that was send in this report should be allowed to send again. Also this message can not be replaced since is never was accepted.

### PR DESCRIPTION
**Pull request by Baxxx**
Closes #267

### CODE REVIEW

**Comment by Baxxx** [comment]
- [x] If the report is replacement for a received report, and it was rejected, you should still be able to resend the original, received report.
- [x] Show fetch feedback only if you didn't do it already (remove it completely once clicked)
- [x] Make it impossible to resend the report without getting the feedback

**Comment by Baxxx** [comment]
- [x] Add column for status (received, replaced, rejected)
- [x] Add column for replaced by and replacement for

**in the view**

File: modules/altinnsalary/view/list.php [hunk 1]
```diff
@@ -132,12 +132,20 @@ function toggleReportDetails(id) {
             <input type="hidden" name="use_work_relation[<? print $work_relation_id; ?>]" value='1'>
           <? } ?>
           <input type="hidden" name="altinnReport1.ExternalShipmentReference" value='<?print 'LODO' . time(); ?>'>
-          <? print $_lib['form3']->submit(array(
-            'name'=>'action_soap1',
-            'value'=>'Send p&aring; nytt',
-            'disabled' => !($so2row->res_ReceiversReference && empty($so1row->ReplacedByMeldindsID))
-          )); ?>
+          <? 
+            $resend_enabled = ($so1row->ReceivedStatus == "received" || empty($so1row->ReceivedStatus)) || ($so2row->res_ReceiversReference && empty($so1row->ReplacedByMeldindsID) && empty($so1row->ReceivedStatus));
```
Comment (Baxxx) [inline modules/altinnsalary/view/list.php]: Remove first `empty()`

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Comment by Baxxx** [comment]
- [x] Instead of having a link that leads to certain altinnReport1, make the link highlight that report on the list.

File: modules/altinnsalary/view/salarylist.php [hunk 2]
```diff
@@ -71,7 +71,7 @@
     if (!($i % 2)) { $sec_color = "BGColorLight"; } else { $sec_color = "BGColorDark"; };
     ?>
     <?
-    $query_report_id  = "SELECT * FROM altinnReport1salary AS ars JOIN altinnReport1 AS ar ON ars.AltinnReport1ID = ar.AltinnReport1ID WHERE ars.SalaryId =  " . $row->SalaryID . " ORDER BY UpdatedAt desc LIMIT 1";
+    $query_report_id  = "SELECT * FROM altinnReport1salary AS ars JOIN altinnReport1 AS ar ON ars.AltinnReport1ID = ar.AltinnReport1ID WHERE ars.SalaryId =  " . $row->SalaryID . " and IFNULL(ReceivedStatus, '') != 'rejected' and IFNULL(ReceivedStatus, '') != 'replaced' ORDER BY UpdatedAt desc LIMIT 1";
```
Comment (Baxxx) [inline modules/altinnsalary/view/salarylist.php]: Use uppercase `AND`

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Pull request merged**